packages/config-service/tests/test.env

```diff
@@ -0,0 +1 @@
+CHAIN_ID=0x160c
```

Comment:
Is this just a random chain id or did you mean for 0x128 for mainnet?

Reply:
It's a random chain id for unit testing purposes only.